music21/stream/base.py

```diff
@@ -6831,26 +6864,16 @@ def makeAccidentals(
                 if e.tie is not None and e.tie.type != 'stop':
                     tiePitchSet.add(e.pitch.nameWithOctave)
 
-                # handle this note's ornaments' otherPitches
-                for orn in e.expressions:
-                    if not isinstance(orn, expressions.Ornament):
-                        continue
-
-                    orn.resolveOtherPitches(e.pitch)
-                    if not orn.otherPitches:
-                        continue
-
-                    orn.updateAccidentalDisplay(
-                        pitchPast=pitchPast,
-                        pitchPastMeasure=pitchPastMeasure,
-                        alteredPitches=alteredPitches,
-                        cautionaryPitchClass=cautionaryPitchClass,
-                        cautionaryAll=cautionaryAll,
-                        overrideStatus=overrideStatus,
-                        cautionaryNotImmediateRepeat=cautionaryNotImmediateRepeat,
-                        lastNoteWasTied=False)
-
-                    pitchPast += orn.otherPitches
+                # handle this note's ornaments' ornamentalPitches
+                self.makeOrnamentalAccidentals(
```

Comment:
just call `makeNotation.makeOrnamentalAccidentals(self, e, ...`

Reply:
There was also a note in the last review against writing this three times -- can it be written once with a helper call, like an embedded secondary function?

Reply:
I tried, but the helper call would need to do different things between calling `e.pitch.updateAccidentalDisplay` and calling `makeNotation.makeOrnamentalAccidentals(e)` in each case.  I couldn't figure out a way to get that right.

But I have moved `makeOrnamentalAccidentals` to makeNotation.py, as requested.

Reply:
Done.

Reply:
okay -- I'll try to get the function compacted myself later.